submissions/BlueLamantine/index.html

```diff
@@ -0,0 +1,677 @@
+                </label>
+                <label for="seat52">
+                  <input
+                    type="checkbox"
+                    id="seat52"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/2"
+                    aria-label="row 5 seat 2"
+                  />
+                  <span class="seats__item"></span>
+                </label>
+                <label for="seat53">
+                  <input
+                    type="checkbox"
+                    id="seat53"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/3"
+                    aria-label="row 5 seat 3"
+                  />
+                  <span class="seats__item"></span>
+                </label>
+                <label for="seat54">
+                  <input
+                    type="checkbox"
+                    id="seat54"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/4"
+                    aria-label="row 5 seat 4"
+                    disabled
+                  />
+                  <span class="seats__item"></span>
+                </label>
+                <label for="seat55">
+                  <input
+                    type="checkbox"
+                    id="seat55"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/5"
+                    aria-label="row 5 seat 5"
+                  />
+                  <span class="seats__item"></span>
+                </label>
+                <label for="seat56">
+                  <input
+                    type="checkbox"
+                    id="seat56"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/6"
+                    aria-label="row 5 seat 6"
+                  />
+                  <span class="seats__item"></span>
+                </label>
+                <label for="seat57">
+                  <input
+                    type="checkbox"
+                    id="seat57"
+                    class="main__checkbox"
+                    name="seat"
+                    value="5/7"
+                    aria-label="row 5 seat 7"
+                  />
+                  <span class="seats__item"></span>
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+        </section>
+        <section class="order">
+          <button
+            type="submit"
+            form="ticketsForm"
+            value="Purchase"
+            class="order__submit"
+          >
+            Purchase
+          </button>
+        </section>
+      </form>
+    </main>
+    <footer>
+      <p class="footer__info">
+        Made by
+        <a href="https://github.com/BlueLamantine" class="ghlink">
+          BlueLamantine
+        </a>
+      </p>
```

Comment:
This alignment a bug or a feature?

![image](https://user-images.githubusercontent.com/16196199/106810314-711f1000-6675-11eb-8eda-b0fc6a5c3721.png)

Reply:
this is a part of my design

Reply:
Ok)